apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -229,18 +235,23 @@ export class ApplicationController {
     existingApplication: Application,
     @Body()
     externalDataDto: PopulateExternalDataDto,
+    @Req() req: Request,
   ): Promise<ApplicationResponseDto> {
     await validateIncomingExternalDataProviders(
       existingApplication as BaseApplication,
       externalDataDto,
     )
-
     const templateDataProviders = await getApplicationDataProviders(
       (existingApplication as BaseApplication).typeId,
     )
+    const headers = (req.headers as unknown) as { authorization?: string }
 
     const results = await callDataProviders(
-      buildDataProviders(externalDataDto, templateDataProviders),
+      buildDataProviders(
+        externalDataDto,
+        templateDataProviders,
+        headers.authorization ?? '',
```

Comment:
Is it ever an acceptable case not to have the auth header here?

Reply:
yes some data providers will not need to use it. I am not sure that the data providers that call the parental leave API will use it actually,